lib/mongo/client.rb

```diff
@@ -942,8 +942,9 @@ def reconnect
     # @option opts [ Session ] :session The session to use.
     # @option opts [ Object ] :comment A user-provided
     #   comment to attach to this command.
-    # @option opts [ Integer | nil ] :timeout_ms Operation timeout in milliseconds.
-    #    Must a positive integer. The default value is unset which means infinite.
+    # @option options [ Integer ] :timeout_ms The operation timeout in milliseconds.
+    #    Must a non-negative integer. An explicit value of 0 means infinite.
```

Comment:
```suggestion
    # @option options [ Integer ] :timeout_ms The operation timeout in milliseconds.
    #    Must be a non-negative integer. An explicit value of 0 means infinite.
```

Reply:
Can you change "Must a non-negative" to "Must be a non-negative" throughout this PR?

Reply:
Did this, good catch, thanks!